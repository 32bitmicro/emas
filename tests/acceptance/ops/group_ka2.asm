BLC 0
EXL 0
BRC 0
NRF 0
