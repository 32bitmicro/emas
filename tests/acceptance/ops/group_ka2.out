@ 0x0000 : 0xe400  /  111 001 0 000 000 000  /  58368
@ 0x0001 : 0xe500  /  111 001 0 100 000 000  /  58624
@ 0x0002 : 0xe600  /  111 001 1 000 000 000  /  58880
@ 0x0003 : 0xe700  /  111 001 1 100 000 000  /  59136
